modules/local/plots/main.nf

```diff
@@ -22,7 +22,7 @@ process PLOTS {
     def prefix = task.ext.prefix ?: "${meta.benchmark_tool}"
 
     """
-    plots.R $summary $prefix
+    plots.R $summary $meta.benchmark_tool
```

Comment:
why does the prefix not work? 🤔

Reply:
that was the main mistake, plots.R takes the name of the benchmarking tool as 2nd command :D my mistake I gave prefix. outputs are generated kind of inside the script..

I have to standardize this script more anyways..